external/easylogging++/easylogging++.cc

```diff
@@ -2202,6 +2202,7 @@ el::base::type::StoragePointer el::base::Storage::getELPP()
 {
   return getresetELPP(false);
 }
+static struct EnsureELPP { EnsureELPP() { el::base::Storage::getELPP(); } } ensureELPP;
```

Comment:
What is the purpose of this? To guarantee initialization? The "initialization order fiasco" is still an issue - this only prevents mistakes if someone forgets to initialize the logger in main. And more importantly, I can't believe I forgot about the mlocker log statement during the "make logging faster" review. It was an obvious culprit for why the change was problematic.